src/Common/DTOs/AddressTypes/Address.cs

```diff
@@ -0,0 +1,19 @@
+namespace SSW.Rewards.Shared.DTOs.AddressTypes;
+
+public class Address
+{
+    public string streetNumber { get; set; }
+    public string streetName { get; set; }
```

Comment:
Should these have proper casing? Or is this being done to bind to JSON? (I think we can use an attribute if that's the case)

Up to you.

```suggestion
    [JsonPropertyName("streetName")]
    public string StreetName { get; set; }
```

Reply:
As per our conversation, we will address this separately. Task captured in #763.